src/projects/diff/hunk-sorter.html

```diff
@@ -160,7 +160,7 @@
         var allGroups = [];
         for (var i = 0, l = ordering.length; i < l; i++) {
           var group = this._reconstructGroup(ordering[i], unsortedHunks);
-          if (group) {
+          if (group && group.diff > 0) {
```

Comment:
`diff` is an array, so the check could better be something like: `group && group.diff && group.diff.length > 0`